src/main/kotlin/org/rust/ide/structure/RustConstTreeElement.kt

```diff
@@ -6,7 +6,7 @@ import org.rust.lang.core.psi.RustConstItemElement
 
 class RustConstTreeElement(element: RustConstItemElement) : PsiTreeElementBase<RustConstItemElement>(element) {
 
-    override fun getPresentableText(): String? = element?.name
+    override fun getPresentableText() = "${element?.name}: ${element?.type?.text}"
```

Comment:
I believe that currently the type is always there, because there are no `pin` atribues associated with const items. However I think that we add `pin`s sooner or later, so the `type` filed would become optional.

It's indeed better to be careful with string formatting, because you can write `"$foo"` even when `foo` is nullable, so you'll get a `"null"` string then.